lib/routes/account.js

```diff
@@ -972,10 +972,6 @@ module.exports = function (
         log.begin('Account.RecoveryEmailResend', request)
         var sessionToken = request.auth.credentials
         var service = request.payload.service || request.query.service
-        if (sessionToken.emailVerified ||
```

Comment:
Although now that you mention it, I wonder if just sending the email again even if verified, is more likely to match user expectations...